gimme_aws_creds/okta.py

```diff
@@ -144,7 +147,7 @@ def auth(self):
         while flow_state.get('apiResponse', {}).get('status') != 'SUCCESS':
             time.sleep(0.5)
             flow_state = self._next_login_step(
-                flow_state.get('stateToken'), flow_state.get('apiResponse'))
+                flow_state.get('apiResponse').get('stateToken'), flow_state.get('apiResponse'))
```

Comment:
Odd... Any idea why this was not a problem before?

Can we change this to `flow_state.get('apiResponse', {}).get('stateToken')` to more gracefully handle a situation where `apiResponse` isn't populated?

Reply:
I'm not sure why it wasn't a problem before. I fixed the code.